tests/test_ofc.py

```diff
@@ -344,6 +343,29 @@ def test_truncate_dof(self):
         self.assertTrue(np.allclose(m1m3_corr(), np.zeros_like(m1m3_corr())))
         self.assertTrue(np.allclose(m2_corr(), np.zeros_like(m2_corr())))
 
+    def test_get_correction(self):
+
+        # First time of calculation
+        correction = self._calculate_m2_hex_correction()
+
+        self.assertAlmostEqual(correction[0], 2.5379271)
+        self.assertAlmostEqual(correction[1], -0.5385152)
+        self.assertAlmostEqual(correction[2], 9.4484754)
+
+        # Second time of calculation
+        # The DOF should be aggregated
+        correction = self._calculate_m2_hex_correction()
+
+        self.assertAlmostEqual(correction[0], 5.2979926)
+        self.assertAlmostEqual(correction[1], -2.5117287)
+        self.assertAlmostEqual(correction[2], 19.6478187)
+
+    def _calculate_m2_hex_correction(self):
```

Comment:
This is another case where I would strongly advocate for not having a private method. These are basically 3 lines of code that are way more comprehensive to have inline in the test method above than having on a private method.

Reply:
After I removed the `self._calculate_corrections()`, this function is 7 lines of code now. Therefore, I will keep it to reduce the duplication. Thanks!